Formula/siril.rb

```diff
@@ -0,0 +1,69 @@
+class Siril < Formula
+  desc "Astronomical image (pre-)processing application"
+  homepage "https://free-astro.org/index.php/Siril"
+  url "https://free-astro.org/download/siril-0.9.8.tar.bz2"
+  sha256 "ecb5477937afc02cc89cb07f4a7b99d2d0ab4cc5e715ec536e9be5c92a187170"
+  head "https://free-astro.org/svn/siril/"
+
+  bottle do
+    sha256 "5c75d117d251a7708d744a09f5c7713e50abe259be256a93f9987c63f4e4881a" => :sierra
+    sha256 "24c5f64cd4377dab1195fa5170a45ca17d9171fb5bb66f6a36f407e4b48bb42e" => :el_capitan
+  end
+
+  option "without-openmp", "Disable OpenMP"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "intltool" => :build
+  depends_on "libtool" => :build
+  depends_on "llvm"
```

Comment:
`  * Dependency 'llvm' may be unnecessary as it is provided by macOS; try to build this formula without it.`

Reply:
It seems you're right.